docs/sharepoint.md

```diff
@@ -11,7 +11,7 @@ In general we want each case's documents to be stored with [principle of least p
 
 ## Environments
 
-We have three Active Directory groups (dev, staging, prod), each with their own SharePoint site/filesystem. Access to one group/site does not grant access to others. Environment access is controlled at the user level by group and folder level permissions (see access section) and at the application level buy these two settings:
+We have three Active Directory groups (dev, staging, prod), each with its own SharePoint site/filesystem. Access to one group/site does not grant access to others. Environment access is controlled at the user level by group and folder level permissions (see access section) and at the application level by these two settings:
```

Comment:
It might be worth mentioning that each Group is tied to an environment (development, staging, production) and this is why we have three of them. The Groups are self contained and independent of each other.